package/gluon-alt-esc-client/luasrc/lib/gluon/web/model/admin/altesc-client.lua

```diff
@@ -0,0 +1,130 @@
+              function(s)
+                 client = s
+                 return false
+              end
+  )
+  return client
+end
+
+local client = get_client(uci)['.name']
+local disabled = uci:get_first('gluon-alt-esc-client', 'client', "disabled")
+
+local site_code
+
+if site.site_code then
+  sitecode = site.site_code
+else
+  sitecode = "ff"
+end
+
+-- reserve space for suffixes, SSID limited to 32 characters by standard
+local ssidlen = 32 - string.len(" #abcd #" .. sitecode)
+local ssiddata = uci:get('wireless', 'altesc_radio0', "ssid")
+
+-- Remove sitecode suffix
+if ssiddata and string.match(ssiddata, " #" .. sitecode .. "$") then
+  ssiddata = string.match(ssiddata, "^(.*) #" .. sitecode .. "$")
+end
+
+-- Remove zone suffix
+if ssiddata and string.match(ssiddata, " #%x%x%x%x$") then
+  ssiddata = string.match(ssiddata, "^(.*) #%x%x%x%x$")
+end
+
+local f = Form(translate("Alternative Exit Service Collaborator - Client"))
+local s = f:section(Section, nil, translate(
+		'This allows you to add a wifi interface with an alternative gateway '
+		 .. 'for its internet connectivity.'
+))
+
+local enabled = s:option(Flag, "enabled", translate("Enabled"))
+enabled.default = ssiddata and disabled and disabled == "0"
+
+local ssid = s:option(Value, "ssid", translate("Name (SSID)"))
+ssid:depends(enabled, true)
+ssid.datatype = "maxlength(" .. ssidlen .. ")"
+ssid.default = ssiddata
+
+local exit4data = uci:get_first('gluon-alt-esc-client', 'client', "exit4")
+local exit4flag = s:option(Flag, "exit4flag", translate("Enable IPv4 redirection"))
+exit4flag:depends(enabled, true)
+exit4flag.default = (exit4data and exit4data ~= "")
+
+local exit4 = s:option(Value, "exit4", translate("Exit ID (MAC)"))
+exit4:depends(exit4flag, true)
+--exit4.datatype = "macaddr"
+exit4.default = exit4data
+
+local exit6data = uci:get_first('gluon-alt-esc-client', 'client', "exit6")
+local exit6flag = s:option(Flag, "exit6flag", translate("Enable IPv6 redirection"))
+exit6flag:depends(enabled, true)
+exit6flag.default = (exit6data and exit6data ~= "")
+
+local exit6 = s:option(Value, "exit6", translate("Exit ID (MAC)"))
+exit6:depends(exit6flag, true)
+--exit6.datatype = "macaddr"
+exit6.default = exit6data
+
+local landata = uci:get_first('gluon-alt-esc-client', 'client', "altesc_on_lan")
+local altesc_on_lan = s:option(Flag, "altesc_on_lan", translate("Enable redirection on LAN"))
+altesc_on_lan:depends(enabled, true)
+altesc_on_lan.default = landata and landata == '1'
+
+function f:write(self, state, data)
+  local client = get_client(uci)['.name']
+
+  uci:set('gluon-alt-esc-client', client, 'disabled', enabled.data and '0' or '1')
+  uci:set('gluon-alt-esc-client', client, 'exit4', exit4.data or '')
+  uci:set('gluon-alt-esc-client', client, 'exit6', exit6.data or '')
+  uci:set('gluon-alt-esc-client', client, 'altesc_on_lan', altesc_on_lan.data and '1' or '0')
+
+  uci:commit('gluon-alt-esc-client')
+
+  i=0
+  util.iterate_radios(
```

Comment:
`iterate_radios()` changed and now needs the UCI config as additional parameter https://github.com/freifunk-gluon/gluon/commit/f48d10bdde46bf67f982115d93ae487b5da7a3e0